packages/url_launcher/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: url_launcher
 
-version: 0.4.2+5
+version: 0.4.3
```

Comment:
Let's bump the version to 1.0.0 (per a previous discussion, our plugins don't appear to be "launched" if their versions are <1.0.0)

Reply:
Ah ok, done